LayoutTests/fast/forms/ios/show-software-keyboard-on-programmatic-dblclick-of-text-field.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html> <!-- webkit-test-runner [ useFlexibleViewport=true ] -->
+<html>
+<meta name="viewport" content="initial-scale=1, width=device-width">
+<head>
+<script src="../../../resources/js-test.js"></script>
+<script src="../../../resources/ui-helper.js"></script>
+<script>
+jsTestIsAsync = true;
+
+addEventListener("load", async () => {
+    description("Tests that a software keyboard is shown when a text field is progammatically double clicked.");
```

Comment:
I think this test currently works by showing the hardware keyboard, rather than the software keyboard.

We could either adjust the description, or add this before calling `UIHelper.doubleActivateAt()`:
```
    await UIHelper.setHardwareKeyboardAttached(false);
```

Reply:
I would also name the test something along the lines of:
> show-software-keyboard-on-programmatic-focus-during-dblclick.html
instead of:
> show-software-keyboard-on-programmatic-dblclick-of-text-field.html

Reply:
Detaching the hardware keyboard seems more intuitive to me -- fixed in https://github.com/WebKit/WebKit/commit/14d54a8425571114d71c3d2cb970d4518877b94a

Reply:
Renamed in the same commit https://github.com/WebKit/WebKit/commit/14d54a8425571114d71c3d2cb970d4518877b94a